.gitignore

```diff
@@ -1,2 +1,5 @@
-asknot.html
+*~
```

Comment:
-1, you should put these in your own global gitignore file (see git's `core.excludesfile` option).